### ISSUE TITLE
Can't delete files with "Confirm file deletion" option enabled

### ISSUE THREAD

**Issue by knopki** [mnaoumov/obsidian-backlink-cache#1]
Steps to reproduce:
 - Load sandbox vault
 - Enable community plugins
 - Install and ~~delete~~ enable `Backlink Cache v1.3.0`
 - Try to delete file from any menu or command palette
 - See error in console

Error:
```
app.js:1 Uncaught (in promise) TypeError: o.count is not a function
    at e.<anonymous> (app.js:1:1486522)
    at app.js:1:237027
    at Object.next (app.js:1:237132)
    at app.js:1:236048
    at new Promise (<anonymous>)
    at v (app.js:1:235793)
    at e.promptForDeletion (app.js:1:1485718)
    at e.<anonymous> (app.js:1:1488063)
    at app.js:1:237027
    at Object.next (app.js:1:237132)
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
v @ app.js:1
e.promptForDeletion @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
(anonymous) @ app.js:1
v @ app.js:1
e.promptForFileDeletion @ app.js:1
(anonymous) @ app.js:1
e.handleEvent @ app.js:1
```

`app.js`:
```javascript
                            r.createEl("p", {
                                cls: "mod-warning",
                                text: rf.dialogue.labelDeleteFolderWarning()
                            })),
                            e instanceof Xy && (o = t.metadataCache.getBacklinksForFile(e),
// exception is on the next line
                            (a = o.count()) > 0 && (s = rf.nouns.linkWithCount({
                                count: a
                            }),
                            l = rf.dialogue.labelExistingBacklink({
                                links: s
                            }),
                            r.createEl("p", {
                                cls: "mod-warning",
                                text: l
                            }))),
```

Debug info:
```
SYSTEM INFO:
	Obsidian version: v1.4.16
	Installer version: v1.4.16
	Operating system: #1-NixOS SMP Wed Jun 28 08:29:53 UTC 2023 5.15.119
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Backlink Cache v1.3.0
```

Workaround: Disable "Confirm file deletion" option in settings.

**Comment by mnaoumov** [comment]
@knopki Thank you. Looking into it

**Comment by mnaoumov** [comment]
@knopki Could not reproduce

https://github.com/mnaoumov/obsidian-backlink-cache/assets/735878/5e8e944d-04af-46ee-8646-a5ce106f0254

What am I missing?

**Comment by knopki** [comment]
@mnaoumov You're missing that I'm very dumb at 2AM. I don't know why I wrote "install and delete". I meant "install and enable".

**Comment by knopki** [comment]
Repro vault https://github.com/knopki/obsidian-backlink-cache-1-repro

**Comment by mnaoumov** [comment]
Fixed in release `1.4.0`
